recipes/tinyspline/all/conanfile.py

```diff
@@ -34,29 +34,22 @@ class TinysplineConan(ConanFile):
     }
 
     def export_sources(self):
-        for p in self.conan_data.get("patches", {}).get(self.version, []):
-            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
```

Comment:
Could you bump required_conan_version to 1.53.0 please?

Reply:
@SpaceIm 
Thanks a lot.
Fixed it.